model/src/main/kotlin/licenses/LicenseView.kt

```diff
@@ -22,6 +22,7 @@ package org.ossreviewtoolkit.model.licenses
 import org.ossreviewtoolkit.model.LicenseSource
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.spdx.SpdxSingleLicenseExpression
+import org.ossreviewtoolkit.spdx.model.LicenseChoice
```

Comment:
Commit message: I had to read the body several times to understand it. For me, a comma after "LicenseView" would have helped.